README.md

```diff
@@ -23,6 +23,7 @@ On top of standard Appium requirements XCUITest driver also expects the followin
 - Xcode and developer tools must be installed. Note, that usually some time is needed for the Appium team to pick up with the support of the most recent Xcode versions, especially beta ones.
 - Connected real devices must be trusted, added to your developer profile and configured properly along with WebDriverAgent signing. Read [Real devices](#real-devices) section _carefully_ to set them up properly before running your tests.
     - Starting from iOS/iPadOS version iOS/iPadOS 16 real devices require enabling of _Developer Mode_ option in _Settings -> Privacy & Security_
+        - [Enabling Developer Mode on a device](https://developer.apple.com/documentation/xcode/enabling-developer-mode-on-a-device)
```

Comment:
Shall we just add a link to the sentence above? Nested lists are harder to scan

Reply:
👍 
I found a few new things as well. Let me update this description